main.go

```diff
@@ -63,13 +63,17 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var enableLeaderElectionId string
 	var probeAddr string
 	var remoteClusterConfigFile string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&enableLeaderElectionId, "leader-elect-id",
```

Comment:
Sorry for being picky, but can you use the same name?
```suggestion
	flag.StringVar(&enableLeaderElectionId, "leader-election-id",
```

Reply:
fixed. thanks